metis-core/metis-core-rest/src/main/java/eu/europeana/metis/core/rest/exception/RestResponseExceptionHandler.java

```diff
@@ -103,4 +104,29 @@ public StructuredExceptionWrapper handleMessageNotReadable(Exception exception,
     return new StructuredExceptionWrapper(
         "Request not readable.\n" + exception.getMessage());
   }
+
+  /**
+   * Handler for specific classes to overwrite behaviour
+   * @param exception the exception thrown
+   * @param response the response that should be updated
+   * @return {@link StructuredExceptionWrapper} a json friendly class that contains the error message for the client
+   */
+  @ExceptionHandler(value = MissingRequestHeaderException.class)
+  @ResponseBody
+  public StructuredExceptionWrapper handleMissingRequestHeaderException(Exception exception,
+      HttpServletResponse response) {
+
+    String header = ((MissingRequestHeaderException) exception).getHeaderName();
+
+    StructuredExceptionWrapper output = new StructuredExceptionWrapper(exception.getMessage());
+    response.setStatus(HttpStatus.BAD_REQUEST.value());
+
+    if(header.equalsIgnoreCase("Authorization")) {
```

Comment:
Not very important, I think good to know is that when comparing to a constant string it's better to use the constant first, like for example so:
"Authorization.equalsIgnoreCase(header)
That is to avoid nullpointerexception

Reply:
That is a good point, will do that